spec/index.bs

```diff
@@ -1266,7 +1266,7 @@ dictionary GPULimits {
         For each possible {{GPUShaderStage}} `stage`,
         the maximum number of {{GPUBindGroupLayoutDescriptor/entries}} for which:
 
-          - {{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/uniform-buffer}}, and
+          - {{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/"uniform-buffer"}}, and
```

Comment:
is that require to surround the variant with quotes? I think we sometimes had that and sometimes not.

Reply:
It's not required, no, but I find it slightly confusing if there's both quoted and unquoted references throughout the doc. The quoted form seems more logical since that's how you have to use it in JS, but ultimately I don't really care which style we pick as long as we're consistent (or have a logical reason why we format it differently in some places.)